uiplib/UIP.py

```diff
@@ -9,7 +9,7 @@
 
 def main():
     settingsParser = ParseSettings()
-    settings = settingsParser.settings
+    settings = settingsParser.get_settings(sys.argv[1:])
```

Comment:
To write tests for `settings.py`, the change had to be made, and this also enables a nice vision of how arguments are being handled.